LizarroAnghela/Collections/src/Circles.java

```diff
@@ -0,0 +1,32 @@
+public class Circles {
+    private double radius;
+    private double diameter;
+
+
+
+    public Circles (double radius, double diameter){
```

Comment:
Constructors doesn't need to initialize all the attributes. For example in this case, there is no need to receive diameter as a param because you can infer it from the radius.
 this.radius = radius;
this.diameter = radius * 2;